autils/archive/ar.py

```diff
@@ -17,10 +17,10 @@
 
 import struct
 
-#: The first eight bytes of all AR archives
```

Comment:
I don't understand why we don't want to document constants.  Removing the `:` makes those docstrings be removed from the generated documentation.  Care to explain?